<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
   background:transparent;
}

QGroupBox
{
background:#2c1e40;
border-style: dotted;
}
QLabel
{
background:transparent;
}</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>401</width>
      <height>561</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>351</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QStackedWidget
{
background:#2c1e40;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>311</width>
         <height>431</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox
{
background:#2c1e40 ;
border-radius:10px;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>28</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
background: #423259;
border-radius:15px;
padding: 25px 25px;
color: #ffffff;
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>350</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background: #310E68 ;
border-radius:7px;
color:#ffffff;
}</string>
        </property>
        <property name="text">
         <string>Profil Resmi Ekle</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border:none;
border-bottom:1px solid  rgba(170,165,178,0.8);
padding: 5px 5px;
color: #ffffff;
}</string>
        </property>
        <property name="placeholderText">
         <string>Adınız</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border:none;
border-bottom:1px solid  rgba(170,165,178,0.8);
padding: 5px 5px;
color: #ffffff;
}</string>
        </property>
        <property name="placeholderText">
         <string>Soyadınız</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border:none;
border-bottom:1px solid  rgba(170,165,178,0.8);
padding: 5px 5px;
color: #ffffff;
}</string>
        </property>
        <property name="placeholderText">
         <string>Kullanıcı Adı</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border:none;
border-bottom:1px solid  rgba(170,165,178,0.8);
padding: 5px 5px;
color: #ffffff;
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string>Şifre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::PreventContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border:none;
border-bottom:1px solid  rgba(170,165,178,0.8);
padding: 5px 5px;
color: #ffffff;
}</string>
        </property>
        <property name="placeholderText">
         <string>2 x 2 =</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>400</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background: transparent ;
border-radius:7px;
color:#ffffff;
}</string>
        </property>
        <property name="text">
         <string>Neden İstiyoruz?</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>400</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background: transparent ;
border-radius:7px;
color:#ffffff;
}</string>
        </property>
        <property name="text">
         <string>Hakkımızda</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>280</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background: #310E68 ;
border-radius:7px;
color:#ffffff;
}</string>
        </property>
        <property name="text">
         <string>ÜYE OL</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>261</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>MSAGE, açık kaynak kodlu, kişilerin kendi sunucularını kurarak veya MSAGE'in hazır sunucularında odalar kurarak mesaj, video, fotoğraf atabildiği aynı zamanda görüntülü sohbet edebildiği bir programdır.  MSAGE'in sunucularını kullanan tüm kullanıcı verileri MSAGE'in kendi bünyesinde saklıdır. Daha fazlası için msage.com adresine bakabilirsiniz.  </string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>HAKKIMIZDA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>330</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background: #310E68 ;
border-radius:7px;
color:#ffffff;
}</string>
       </property>
       <property name="text">
        <string>Üye Ol</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>NEDEN İSTİYORUZ?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>330</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background: #310E68 ;
border-radius:7px;
color:#ffffff;
}</string>
       </property>
       <property name="text">
        <string>Üye Ol</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>261</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>MSAGE içerisinde olası acil bir durumda kullanıcılara ulaşmak, gerektiğinde özel olarak bilgi vermek amacıyla toplanan tüm veriler MSAGE'in güvenli veri tabanlarında saklanmaktadır.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background: #3c031c;
color:#ffffff;
border-radius:7px;
border-style: dotted;
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway Black</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
background:transparent;
color:#ffffff;
}</string>
     </property>
     <property name="text">
      <string>MSAGE</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
